src/netbase.h

```diff
@@ -37,6 +38,79 @@ class proxyType
     bool randomize_credentials;
 };
 
+/**
+ * Convert milliseconds to a struct timeval for e.g. select.
+ */
+struct timeval MillisToTimeval(int64_t nTimeout);
```

Comment:
This move is an artifact of an earlier commit

Reply:
Now addressed. Thanks!